ptvsd/ipcjson.py

```diff
@@ -352,7 +355,9 @@ def on_event(self, msg):
     def on_invalid_request(self, request, args):
         # TODO: docstring
         self.send_response(request, success=False, message='Unknown command')
-        if self.__exit_on_unknown_command:
+        if self._exit_on_unknown_command:
+            # TODO: Shouldn't we let VSC decide how to handle this
```

Comment:
I think fast-fail is a reasonable approach. If the other side sends an unknown command, it may well be a request that requires a reply, and will block the IDE until it receives one. But it would be tricky to diagnose if the message is simply ignored.

Reply:
We always send a response message (with success=False).  So the IDE will know that the request failed.